flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/sink/filesystem/Bucket.java

```diff
@@ -230,6 +256,11 @@ void onSuccessfulCompletionOfCheckpoint(long checkpointId) throws IOException {
 
 	void rollOnProcessingTimeIfNeeded(long timestamp) throws IOException {
 		if (inProgressPart != null && rollingPolicy.shouldRollOnProcessingTime(inProgressPart, timestamp)) {
+			if (LOG.isDebugEnabled()) {
+				LOG.debug("Subtask {} closing in-progress part file for bucket id={} due to timeout " +
```

Comment:
"timeout" has typically a negative connotation. The exact semantics here are dependent on the user-defined policy, so I would just say "due to processing time policy".